inspirehep/modules/theme/jinja2filters.py

```diff
@@ -590,6 +590,12 @@ def is_external_link(url):
             ('.pdf', '.png', '.jpg', '.jpeg')))
 
 
+@blueprint.app_template_filter()
+def is_institute(institute):
+    """Checks if given string is an institute."""
+    return institute.lower() in ['kekscan', 'ads', 'cds', 'hal', 'msnet',
+                                 'osti', 'zblatt', 'euclid']
```

Comment:
Wrong rebase: this died in https://github.com/inspirehep/inspire-next/commit/e35c380ba093ca6a78e71b5443d6c6cbc21bbf46, it should stay dead.